### ISSUE TITLE
Question: Can markdig be customized to output dita files used for technical documenation? Attached a sample conversion in the comments

### ISSUE THREAD

**Issue by bordoloiworks** [xoofx/markdig#691]
![MD2Dita](https://user-images.githubusercontent.com/76158868/212287692-8874c624-9111-45ad-9b97-21783c72f051.png)

**Comment by gfoidl** [comment]
Should be possible with custom [renderers](https://github.com/xoofx/markdig/tree/master/src/Markdig/Renderers).
But quite a bit of work...so I'd go with something like [markdown2dita](https://github.com/mattcarabine/markdown2dita) (even if it's not a .NET-based solution, but for simplicity one shouldn't be that strict 😉).

**Comment by xoofx** [comment]
> Can markdig be customized to output dita files used for technical documenation? Attached a sample conversion in the comments 

Yes, likely, otherwise, as @gfoidl is suggesting, it's also quite some work. It's up to your project/requirements.
